bin/fastlane

```diff
@@ -4,6 +4,30 @@ if RUBY_VERSION < '2.0.0'
   abort("fastlane requires Ruby 2.0.0 or higher")
 end
 
+def self.windows?
+  # taken from: https://stackoverflow.com/a/171011/1945875
+  (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+end
+
+require 'bootsnap'
+require 'tmpdir'
+require 'fileutils'
+# env = ENV['RAILS_ENV'] || "development"
```

Comment:
This can probably get 💣 ed?

Reply:
Can we get rid of this comment? The `RAILS_ENV` example is not relevant to _fastlane_ at all